jekyll/_cci2/ssh-access-jobs.md

```diff
@@ -18,19 +18,23 @@ This document describes how to access a build container using SSH on CircleCI 2.
 Often the best way to troubleshoot problems is to SSH into a build container and inspect 
```

Comment:
Maybe "is to SSH into a build container" -> "is to SSH into a Job"

My reasoning, a Job can run in a container for the `docker` executor but a VM for the `machine` or `macos` executors. It's not always containers.